sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/BooleanSimplificationSuite.scala

```diff
@@ -37,6 +38,7 @@ class BooleanSimplificationSuite extends PlanTest with PredicateHelper {
       Batch("Constant Folding", FixedPoint(50),
         NullPropagation(conf),
         ConstantFolding,
+        SimplifyConditionals,
         BooleanSimplification,
         PruneFilters(conf)) :: Nil
   }
```

Comment:
At line 46, SPARK-17851 changed like the following. We need that.
```scala
- val testRelation = LocalRelation('a.int, 'b.int, 'c.int, 'd.string)
+ val testRelation = LocalRelation('a.int, 'b.int, 'c.int, 'd.string,
   'e.boolean, 'f.boolean, 'g.boolean, 'h.boolean)
```

Reply:
I see. Thanks!